tests/unit/autoprompt/test_doc.py

```diff
@@ -96,16 +96,16 @@ def test_get_doc_content(self):
 
     def test_get_rst_doc_in_txt(self):
         content = textwrap.dedent("""\
-            MySection
```

Comment:
It would be nice if we break this test into its own separate test case (e.g. `get_doc_content_strips_carriage_returns`). My main concern in reusing the existing test case is that it does not indicate that it also ensures  carriage returns are stripped out. So if we ever update the content, it may be easy to overlook that the carriage returns need to be retained for the test.